hpedockerplugin/volume_manager.py

```diff
@@ -362,6 +364,10 @@ def clone_volume(self, src_vol_name, clone_name,
 
         if not size:
             size = src_vol['size']
+        if not cpg:
+            cpg = src_vol['cpg']
```

Comment:
Check this condition for upgrade scenario

Reply:
Done